### DISCUSSION TITLE
Support template in send email operation

### DISCUSSION THREAD

**Discussion by massimans** [directus/directus#14091] [Feature Requests]
Hi,
currently the send email operation supports only sending raw email (with some markdown formatting)

```javascript
await mailService.send({
    html: md(body),
    to,
    subject,
});
```
it would be nice to have the possibility to activate the base layout with a checkbox for example

**Comment by massimans** [comment]
i can contribute with a pull request if you think it's a good idea since its a trivial change

**Reply by rijkvanzanten** [reply]
Sounds good to me!

**Reply by massimans** [reply]
Greate,i really like what you all doing with the project and was thinking of contributing for some time now,this could be a good starting point, i already scouted the code and i think can do it , but i would need some guidance on how to setup a dev environment

**Reply by rijkvanzanten** [reply]
This should get you started: https://docs.directus.io/contributing/running-locally/

**Reply by massimans** [reply]
Hi,
the app from main branch don't seem to build correctly vite complaints about the .md loader
`No loader is configured for ".md" files: ../node_modules/@directus/docs/self-hosted/installation/plesk.md `
but works as expected in V9.12.2.
i tried it both in win11 and wsl ubuntu with node installed from volta, the diff for vite.config.js  
```git
diff --git a/app/vite.config.js b/app/vite.config.js
index 210fe2d3f..b3af0f701 100644
--- a/app/vite.config.js
+++ b/app/vite.config.js
@@ -1,6 +1,6 @@
 import { defineConfig, searchForWorkspaceRoot } from 'vite';
 import vue from '@vitejs/plugin-vue';
-import md from 'vite-plugin-md';
+import md from 'vite-plugin-vue-markdown';
 import yaml from '@rollup/plugin-yaml';
 import path from 'path';
 import {
@@ -23,7 +23,7 @@ export default defineConfig({
                        include: [/\.vue$/, /\.md$/],
                }),
                md({
-                       wrapperComponent: 'DocsWrapper',
+                       wrapperComponent: 'docs-wrapper',
                        markdownItOptions: {
                                highlight(str, lang) {
                                        if (lang && hljs.getLanguage(lang)) {
```
shows that indeed changes were made to the md loader between the version that may have broke something

any suggestions ?

**Reply by rijkvanzanten** [reply]
That was resolved in https://github.com/directus/directus/pull/14214 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
